src/main/kotlin/org/elm/ide/test/run/ElmTestRunProfileState.kt

```diff
@@ -83,14 +89,22 @@ class ElmTestRunProfileState internal constructor(
     }
 
     override fun createConsole(executor: Executor): ConsoleView? {
+        if (elmProject == null)
```

Comment:
just make this one line: `if (elmProject == null) error("Elm project not found")`